docs/core/run-time-config/garbage-collector.md

```diff
@@ -0,0 +1,204 @@
+| **app.config for .NET Framework** | [GCHeapAffinitizeMask](../../framework/configure-apps/file-schema/runtime/gcheapaffinitizemask-element.md) | *decimal value* | 4.6.2 |
+
+### System.GC.GCHeapAffinitizeRanges/COMPlus_GCHeapAffinitizeRanges
+
+- Specifies the list of processors to use for garbage collector threads.
+- This setting is similar to `System.GC.HeapAffinitizeMask`, except it allows you to specify more than 64 processors.
+- For Windows operating systems, prefix the processor number or range with the corresponding [CPU group](/windows/win32/procthread/processor-groups), for example, "0:1-10,0:12,1:50-52,1:70".
+- If processor affinity is disabled by setting `System.GC.NoAffinitize` to `true`, this setting is ignored.
+- Applies to server garbage collection (GC) only.
+- For more information, see [Making CPU configuration better for GC on machines with > 64 CPUs](https://devblogs.microsoft.com/dotnet/making-cpu-configuration-better-for-gc-on-machines-with-64-cpus/) on Maoni Stephens' blog.
+
+| | Setting name | Values | Version introduced |
+| - | - | - | - |
+| **runtimeconfig.json** | `System.GC.GCHeapAffinitizeRanges` | Comma-separated list of processor numbers or ranges of processor numbers.<br/>Unix example: "1-10,12,50-52,70"<br/>Windows example: "0:1-10,0:12,1:50-52,1:70" | .NET Core 1.0 |
+| **Environment variable** | `COMPlus_GCHeapAffinitizeRanges` | Comma-separated list of processor numbers or ranges of processor numbers.<br/>Unix example: "1-10,12,50-52,70"<br/>Windows example: "0:1-10,0:12,1:50-52,1:70" | .NET Core 1.0 |
+| **app.config for .NET Framework** | N/A | N/A | N/A |
+
+### COMPlus_GCCpuGroup
+
+- Configures whether the garbage collector uses [CPU groups](/windows/win32/procthread/processor-groups) or not.
+
+  When a 64-bit Windows computer has multiple CPU groups, that is, there are more than 64 processors, enabling this element extends garbage collection across all CPU groups. The garbage collector uses all cores to create and balance heaps.
+
+- Applies to server garbage collection (GC) on 64-bit Windows operation systems only.
+- Default: Disabled (0).
+- For more information, see [Maoni Stephens' blog entry](https://devblogs.microsoft.com/dotnet/making-cpu-configuration-better-for-gc-on-machines-with-64-cpus/).
+
+| | Setting name | Values | Version introduced |
+| - | - | - | - |
+| **runtimeconfig.json** | N/A | N/A | N/A |
+| **Environment variable** | `COMPlus_GCCpuGroup` | 0 - disabled<br/>1 - enabled | .NET Core 1.0 |
+| **app.config for .NET Framework** | [GCCpuGroup](../../framework/configure-apps/file-schema/runtime/gccpugroup-element.md) | `false` - disabled<br/>`true` - enabled |  |
+
+### System.GC.NoAffinitize/COMPlus_GCNoAffinitize
+
+- Specifies whether to affinitize garbage collection threads with processors. That is, whether to create a dedicated heap, GC thread, and background GC thread (if background garbage collection is enabled) for each processor.
```

Comment:
BGC threads are actually not affinitized.
for Server GC threads, we should just say that the GC thread is affinitized. "a dedicated heap" could be interpreted as heaps cannot point to each other but in fact they can. even if the GC threads are not affinitized, we still create a heap per GC thread. so perhaps something like -

"A heap is created for each GC thread; affinity means a GC thread can only run on its specific CPU"